PolicyDecisionPoint/src/main/java/it/cnr/iit/ucs/pdp/PolicyDecisionPoint.java

```diff
@@ -81,11 +78,11 @@
     private static Logger log = Logger.getLogger( PolicyDecisionPoint.class.getName() );
 
     private PDPConfig pdpConfig;
-    private Journal journal = null;
+    private Optional<JournalingInterface> journalInterface;
 
     public PolicyDecisionPoint( PdpProperties properties ) {
         super( properties );
-        buildJournal( properties.getJournalDir() );
+        journalInterface = it.cnr.iit.ucs.journaling.JournalBuilder.build( properties );
```

Comment:
why do you need to declare it with full package definition here

Reply:
yes, I noticed it imediately after the pull request, if you update you should not see it anymore